web/ui/dashboard/src/app/private/components/create-app/create-app.component.html

```diff
@@ -63,16 +58,21 @@ <h2 class="font-semibold">Endpoint {{ i > 0 ? i + 1 : '' }}</h2>
 					</div>
 				</div>
 			</div>
-			<convoy-button (click)="addEndpoint()" type="text" className="px-0 font-medium">
-				<img src="/assets/img/add-icon.svg" alt="close icon" class="mr-2" />
+			<convoy-button (click)="addEndpoint()" type="text" className="px-0 font-medium text-14">
+				<img src="/assets/img/add-icon.svg" alt="close icon" class="mr-2 h-16px" />
 				Add Endpoint
 			</convoy-button>
 		</div>
 	</convoy-card>
 	<div class="flex justify-end mt-6 mb-16">
-		<convoy-button buttonType="button" (click)="closeAppInstance()" type="outline" className="mr-3">Discard</convoy-button>
+		<convoy-button buttonType="button" (click)="confirmModal = true" type="outline" className="mr-3 transition-all duration-300 hover:text-danger-100 hover:border-danger-100">Discard</convoy-button>
 		<convoy-button [disable]="isSavingApp" (click)="saveApp()">
-			{{ editAppMode ? 'Edit App' : 'Create App' }}
+			{{ isNewProjectRoute() ? 'Save and Proceed' : editAppMode ? 'Edit App' : 'Create App' }}
```

Comment:
@Oluwadaminiola What do you think about this being a pipe instead of a method duplicated almost everywhere?